cherrypy/tutorial/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3-alpine
+
+COPY . /tutorial
+
+WORKDIR /tutorial
+
+RUN pip install cherrypy && \
+  sed -i 's/^server\.socket_host.*$/server.\socket_host = "0.0.0.0"/' \
```

I believe, there's a typo here:
```suggestion
  sed -i 's/^server\.socket_host.*$/server.socket_host = "0.0.0.0"/' \
```